src/util.c

```diff
@@ -420,7 +442,11 @@ int write_result_into_log_file(struct ntttcp_test_endpoint *tep)
 	if (tep->endpoint_role == ROLE_SENDER && test->protocol == TCP) {
 		fprintf(logfile, "	<tcp_average_rtt metric=\"us\">%u</tcp_average_rtt>\n", tepr->average_rtt);
 	}
-
+	
+	p=malloc(256);
+	memset(p, '\0', sizeof(char) * 256);
+	count = execute_system_cmd_by_process("uname -a", "r", p);
```

Comment:
Why we need this line, and where did you use it.

Reply:
I found that I can't get the correct address by using array so it will cause incomplete information to be output, but using "malloc" can get, so I added a var to get the correct address in order to output complete information.